test/test_tree_2d_euler.jl

```diff
@@ -110,35 +110,40 @@ EXAMPLES_DIR = joinpath(pathof(Trixi) |> dirname |> dirname, "examples", "tree_2
     @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_euler_blast_wave_pure_fv.jl"),
       l2   = [0.39957047631960346, 0.21006912294983154, 0.21006903549932, 0.6280328163981136],
       linf = [2.20417889887697, 1.5487238480003327, 1.5486788679247812, 2.4656795949035857],
-      tspan = (0.0, 0.5))
+      tspan = (0.0, 0.5),
+      coverage_override = (maxiters=10^5, tspan=(0.0, 0.1)))
```

Comment:
Why do you use a big `maxiters` here?

Reply:
Again, AMR case that needs to run longer than a certain physical time.